packages/contracts-bedrock/contracts/L1/L1StandardBridge.sol

```diff
@@ -100,6 +100,17 @@ contract L1StandardBridge is StandardBridge, Semver {
         StandardBridge(_messenger, payable(Predeploys.L2_STANDARD_BRIDGE))
     {}
 
+    /**
+     * @notice Allows EOAs to deposit ETH by sending directly to the bridge.
+     */
+    receive() external payable override onlyEOA {
+        // Notice this emits an `ETHDepositInitiated` event.
+        // Whereas `StandardBridge` contract's `receive` fallback will only call
+        // `_initiateBridgeEth`, which will emit an `ETHBridgeInitiated` event.
+        emit ETHDepositInitiated(msg.sender, msg.sender, msg.value, bytes(""));
+        _initiateBridgeETH(msg.sender, msg.sender, msg.value, RECEIVE_DEFAULT_GAS_LIMIT, bytes(""));
```

Comment:
I'm wondering if emitting it inside of `_initiateBridgeETH` would make this less bug prone in the future, having the least amount of places where it is emitted using shared code paths

Reply:
I agree with @tynes that we want to write it in as few places as possible, but the challenge is that if we put the `ETHDepositInitiated` event into `_initiateBridgeETH()` it would then also be emitted [here](https://github.com/ethereum-optimism/optimism/blob/5ed518a0ea7334d6e6448104a005a094ae4dbbb7/packages/contracts-bedrock/contracts/L2/L2StandardBridge.sol#L172) by the `_initiateWithdrawal()` function. 

Here's one proposal to make this work:
Modify the internal bridging functions (`_initiateBridgeETH()` and `_initiateBridgeERC20()` to take an extra argument (maybe `enum Direction { Deposit, Withdrawal }`) and use that to determine which events to emit.